docs/_toc.yml

```diff
@@ -8,8 +8,15 @@
 #   - file: introduction
 
 - part: Plotting
-  chapters: 
+  chapters:
   - file: plotting_3d
-- part : Summary Statistics
-  chapters: 
-  - file: GraphStats
\ No newline at end of file
+  - file: plotting_Adj
+- part : Analysis
+  chapters:
+  - file: connectome_overview
+  - file: apriori_sbms
+  - file: GraphStats 
+  - file: Modularity
+- part: Affinity Testing
```

Comment:
This should be like the others: just everyone's notebook is a different chapter in analysis

Reply:
OK